orcid-model/src/main/java/org/orcid/jaxb/model/v3/release/record/Spam.java

```diff
@@ -0,0 +1,122 @@
+package org.orcid.jaxb.model.v3.release.record;
+
+import java.io.Serializable;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+import javax.xml.bind.annotation.XmlType;
+
+import org.orcid.jaxb.model.v3.release.common.CreatedDate;
+import org.orcid.jaxb.model.v3.release.common.LastModifiedDate;
+import org.orcid.jaxb.model.v3.release.common.ReportedDate;
+
+import io.swagger.annotations.ApiModel;
+
+/**
+ * 
+ * @author Daniel Palafox
+ * 
+ */
+@XmlAccessorType(XmlAccessType.FIELD)
+@XmlType(propOrder = { "createdDate", "lastModifiedDate", "sourceType", "spamCounter", "reportedDate" })
+@XmlRootElement(name = "spam", namespace = "http://www.orcid.org/ns/spam")
+@ApiModel(value = "SpamV3_0")
+public class Spam implements Serializable {
+    private static final long serialVersionUID = 1L;
+
+    @XmlElement(name = "source-type", namespace = "http://www.orcid.org/ns/spam")
```

Comment:
@DanielPalafox you don't need the XML annotations since this is not returned in the API

Reply:
@amontenegro I added this because perhaps we can use the API to automate the process such as getting records that have been reported more than 10 times etc.